RabbitMQ.Stream.Client/Client.cs

```diff
@@ -239,15 +239,11 @@ public static async Task<Client> Create(ClientParameters parameters, ILogger log
 
         public async ValueTask<bool> Publish(Publish publishMsg)
         {
-            var publishTask = Publish<Publish>(publishMsg);
-            if (!publishTask.IsCompletedSuccessfully)
```

Comment:
For more info about this change, see:
 https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2012